### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by HansZizold**
** CHANGES **
Styles added to the project to match Zeplin design.

### CODE REVIEW

File: src/Components/Books.module.css [hunk 1]
```diff
@@ -1,14 +1,11 @@
-.title {
-  margin: 80px 16% 0;
-  border: 1px solid black;
-  width: 50vw;
+.books {
+  background-color: aliceblue;
```
Comment (andersonlebon) [inline src/Components/Books.module.css]: - [ ] Kindly remove this background color as the requirements say that your design should match the original template

image

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @HansZizold 


Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Components/Books.module.css (hunk 1 above)
Comment (HansZizold) [reply src/Components/Books.module.css]: Hi @andersonlebon, I checked the background color in Zeplin and I changed it as you requested.

**Comment by HansZizold** [comment]
Hi @andersonlebon, I changed the background color according to Zeplin design.
Thank you very much for your review.

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey @HansZizold  😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by HansZizold** [comment]
@andersonlebon, thank you very much!!!

**Pull request merged**